DEVELOPMENT.md

```diff
@@ -155,6 +155,8 @@ This step includes building Knative Serving, creating and pushing developer imag
 First, edit [config-network.yaml](config/config-network.yaml) as instructed within the file.
 If this file is edited and deployed after Knative Serving installation, the changes in it will be
 effective only for newly created revisions.
+Alternatively, if you are developing on GKE, you can choose not to edit the file and use the patching tool
+in `hack/dev-patch-config-gke.sh`.
```

Comment:
the script already apply the updated config, so what's in "Next" no longer applies.

Reply:
Maybe we can make the script show the modification before applying and ask for confirmation

Reply:
yes, even better.

Reply:
Done